Lib9c/TableData/Skill/StatBuffSheet.cs

```diff
@@ -33,8 +33,11 @@ public class Row : SheetRow<int>
             public int Duration { get; private set; }
 
             public SkillTargetType TargetType { get; private set; }
-            public StatModifier StatModifier { get; private set; }
-            public string IconResource { get; private set; }
```

Comment:
is this correct change?

Reply:
`IconResource` is not used in state, it's parse is replaced with `TryParse`. And `StatModifier` field is replaced into this.
```
            public int Value { get; private set; }
            public StatType ReferencedStatType { get; private set; }
```